### DISCUSSION TITLE
UV-K5(8) speaker volume

### DISCUSSION THREAD

**Discussion by chaffinched** [armel/uv-k5-chirp-driver#32] [General]
I've just bought one of these, installed the latest F4HWN firmware (thanks for your hard work!) and I'm gradually finding my way around. The main issue for me at the moment is that the volume knob goes from Off to "way too loud" on the after the smallest of turns, and just gets louder from there. 

Before I implement a hardware fix, is there anything I can do in the firmware? I see a "DAC Gain" and "Volume Gain" in the calibration, are these safe to play with or are they unrelated?

**Comment by armel** [comment]
Hello,

Thank you for your message.
Yes, you can play with Volume Gain. Default value is 44. 
You can choose a value between 0 and 63 (Volume Gain set to 6 bits). 
Remember to make a backup of your calibration eeprom before testing, just in case ;)

Armel.

**Comment by chaffinched** [comment]
Thanks, that's helped. I had to take Volume Gain down to 5 which makes the beeps too loud, but they can be turned off, so all is good!

**Reply by armel** [reply]
My pleasure 😌
